server/game/cards/07-GR/Kaspara.js

```diff
@@ -0,0 +1,32 @@
+const Card = require('../../Card.js');
+
+class Kaspara extends Card {
+    // X is the combined total of between each player’s pools.
+    //
+    // Each friendly Geistoid creature gains, “Play/Destroyed: Each player gains 1.”
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            effect: ability.effects.modifyPower(
+                (card) =>
+                    card.controller.amber +
+                    (card.controller.opponent ? card.controller.opponent.amber : 0)
+            )
+        });
+
+        this.persistentEffect({
```

Comment:
Each friendly create needs to gain a. "Play" effect as well.

Reply:
Good catch, done, thanks.